cellbrowser_tools/fov_processing.py

```diff
@@ -435,14 +425,71 @@ def _generate_paths(self):
         self.atlas_dir = atlasdir
 
     def build_file_list(self):
+        # TODO:
+        # make all this data driven.
+        # either we just take the ALL channels from the images IN ORDER
+        # or we provide some kind of listing and renaming:
+        #     source file
+        #     channel index
+        #     new name
+        #     color (maybe not even needed)
+        # this is enough data to combine together many channels from many files
+
         # start with 4 channels for membrane, structure, nucleus and transmitted light
-        self.channel_names = ["OBS_Memb", "OBS_STRUCT", "OBS_DNA", "OBS_Trans"]
+        self.channel_names = ["Membrane", "Labeled structure", "DNA", "Bright field"]
```

Comment:
indicate that these are specific to a particular data set, or maybe branch to keep the old ones around.  Best solution: make this data driven